renderer/src/styles/ui/bdsettings.css

```
@@ -165,7 +165,7 @@
 }
 
 .bd-settings-title {
-    color: #FFFFFF;
+    color: var(--header-primary);
```

I'd rather have a fallback here.
```suggestion
    color: var(--header-primary, #FFFFFF);
```